AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -276,6 +276,7 @@ consent-pref.trustarc.com#%#//scriptlet('trusted-click-element', 'div[aria-label
 !
 ! NOTE: Regular rules
 !
+merge.dev##.gdpr-cookie-banner
```

Comment:
Can be added into `surveymonkey.de,petri.com,surveymonkey.com,canadapost.ca,research.net,thurrott.com##.gdpr-cookie-banner`.